examples/src/main/scala/sttp/tapir/examples/PrometheusMetricsExample.scala

```diff
@@ -0,0 +1,87 @@
+package sttp.tapir.examples
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives.concat
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.auto._
+import io.prometheus.client.{CollectorRegistry, Counter}
+import sttp.monad.FutureMonad
+import sttp.tapir._
+import sttp.tapir.generic.auto._
+import sttp.tapir.json.circe._
+import sttp.tapir.metrics.Metric
+import sttp.tapir.metrics.prometheus.PrometheusMetrics
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.akkahttp.{AkkaHttpServerInterpreter, AkkaHttpServerOptions}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+object PrometheusMetricsExample extends App with StrictLogging {
+
+  case class Person(name: String)
+
+  // Simple endpoint returning 200 or 400 response with string body
+  val personEndpoint: ServerEndpoint[Person, String, String, Any, Future] =
+    endpoint.post
+      .in("person")
+      .in(jsonBody[Person])
+      .out(stringBody)
+      .errorOut(stringBody)
+      .serverLogic { p =>
+        Future.successful {
+          Thread.sleep(10000 + (math.random() * 100).toInt)
+          if (p.name == "Jacob") Right("Welcome") else Left("Unauthorized")
+        }
+      }
+
+  implicit val monad: FutureMonad = new FutureMonad()
```

Comment:
our goal should be to be able to remove this from the example - that shouldn't be necessary for users to use the metrics (if possible - maybe not?)

Reply:
removed